microcosm_elasticsearch/tests/test_registry.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test registry management.
+
+"""
+from hamcrest import assert_that, equal_to, is_
+from microcosm.api import create_object_graph
+
+from microcosm_elasticsearch.registry import IndexRegistry
+
+
+def test_name_for():
```

Comment:
It would, but I'm not ready to commit to the index interface until I test a few more index operations